navigation/AppNavigator/SettingsNavigator.tsx

```diff
@@ -25,10 +25,7 @@ type Props = AppNavigatorNavigationData<'Settings'>;
 const SettingsNavigator: React.FC<Props> = () => {
   return (
     <SettingsStackNavigator.Navigator screenOptions={defaultScreenOptions}>
-      <SettingsStackNavigator.Screen
-        name="Settings"
-        component={SettingsScreen}
-      />
+      <SettingsStackNavigator.Screen name="Settings" component={AuthScreen} />
```

Comment:
what did happen here ? :thinking:

Reply:
You can log in in the initial screen and then I allowed to log out via Settings tab :D have a look.